src/manifest_graph

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python3 -u
+
+'''
+    Create a tree chart with all the manifest includes
+    symlinks are displayed with a (L) prefix
+'''
+
+import sys
+import yaml
+import os
+from anytree import Node, RenderTree
+
+def collect_included_files(filename, parent):
+    node = None
+
+    if os.path.islink(filename):
+      node = Node("(L)"+filename, parent=parent)
+      dest = os.readlink(filename)
+      collect_included_files(dest, parent=node)
+      return node
+    else:
+      node = Node(filename, parent=parent)
+
+    with open(filename, 'r') as file:
+        try:
+            data = yaml.safe_load(file)
+            if data is None:
+                return
+            include_paths = data.get('include')
```

Comment:
Note in the generic case, there's also `conditional-include` and `arch-include`. Probably worth marking those differently too.